apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/BrevModel.kt

```diff
@@ -18,24 +19,45 @@ enum class Status {
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 data class Adresse(
-    val navn: String? = null,
-    val adresse: String? = null,
+    val adresseType: String,
```

Comment:
Kan adressetype vera ein enum?

Reply:
Problemet nå er at vi må håndtere både regoppslag og dokdist sin adressetype, som for sikkerhets skyld er ulike.. Skal se om jeg finner på noe lurt.